### DISCUSSION TITLE
你好

### DISCUSSION THREAD

**Discussion by lish1609** [Potterli20/hosts#21] [General]
请问可以使用你的规则，加到自己的dns黑名单里面吗。

**Comment by Potterli20** [comment]
> 请问可以使用你的规则，加到自己的dns黑名单里面吗。

你是用在那里？

**Reply by lish1609** [reply]
adguard home里面的dns黑名单

**Reply by Potterli20** [reply]
> adguard home里面的dns黑名单

可以的，但是建议用在x86，还有内存最好给2g以上

**Reply by lish1609** [reply]
谢谢
